src/main/java/com/amihaiemil/versioneye/VersionEye.java

```diff
@@ -47,4 +49,11 @@
      * @return Users.
      */
     Users users();
+    
+    /**
+     * VersionEye with trusted wire.
+     * @return VersionEye.
+     * @throws IOException 
```

Comment:
@SherifWaly ``@throws IOException If something goes wrong with the HTTP call`` and can you also add ``@see http://http.jcabi.com/pkix-validator.html``, so the user can go and read he might need this method :D